<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QWidget" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TouchControl for Android v1.0.1 QT5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ported to Linux by Bj√∂rn Ternes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Orginally written by k.janku1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="github">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>GitHub</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bternescom">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>bternes.com</string>
   </property>
  </widget>
  <widget class="QPushButton" name="github_orginal">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>GitHub</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>This software use droidAtScreen written by Jens Riboe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="droid_website">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Website</string>
   </property>
  </widget>
  <widget class="QPushButton" name="github_droid">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>GitHub</string>
   </property>
  </widget>
  <widget class="QLabel" name="img">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
